Resources/Storage/StorageFactory.py

```diff
@@ -11,6 +11,7 @@
     getStorages()      This takes a DIRAC SE definition and creates storage stubs for the protocols found in the CS.
                       By providing an optional list of protocols it is possible to limit the created stubs.
 """
+from __builtin__ import False
```

Comment:
Yes, Eclipse is now dong very strange things if one's not careful like adding unwanted lines or strings... Not clear if this is a bad setting or effect of a new version